File: initialCode/triangle.py
import math

a = float(input("What is side A?"))
b = float(input("What is side B?"))
c = float(input("What is side C?"))

# Semi perimeter
s = (a + b + c)/2

# Area
v = s * (s - a) * (s - b) * (s - c)

A = math.sqrt(v)

print(A)

File: initialCode/triangle.c
#include<stdio.h>
#include<math.h>

int main() {
    float a;
    float b;
    float c;
    printf("What is side A?");
    scanf("%f", &a);
    printf("What is side B?");
    scanf("%f", &b);
    printf("What is side C?");
    scanf("%f", &c);

    float s = (a + b + c)/2;

    // Area
    float A = sqrt(s * (s - a) * (s - b) * (s - c));

    printf("%f", A);

}